### ISSUE TITLE
chmod: cannot access '/etc/systemd/system/docker.service.d/http-proxy.conf': No such file or directory

### ISSUE THREAD

**Issue by Igonato** [gantsign/development-environment#598]
The machine is booting and everything seems to work just fine, there is just this error message after vagrant up:
```
...
==> default: Machine booted and ready!
==> default: Configuring proxy for Apt...
==> default: Configuring proxy for Docker...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

chmod 0644 /etc/systemd/system/docker.service.d/http-proxy.conf

Stdout from the command:



Stderr from the command:

chmod: cannot access '/etc/systemd/system/docker.service.d/http-proxy.conf': No such file or directory
```
Vagrant 2.2.2, Virtualbox 5.2.4

I have proxy.enabled set to false (as by default)

I see `/etc/systemd/system/docker.service.d/proxy.conf` referenced in the `angstwad.docker_ubuntu` role, not http-proxy.conf. Can't find anything related in the main config  though, can use some help there

**Comment by freemanjp** [comment]
@Igonato thanks for reporting this issue.

It appears this error is from the `vagrant-proxyconf` plugin. It looks related to https://github.com/tmatilai/vagrant-proxyconf/issues/172, it seems to have broken in the the [2.0.2](https://github.com/tmatilai/vagrant-proxyconf/releases/tag/v2.0.2) release.

As a temporary workaround, you'll need to manually downgrade to the 2.0.1 release of `vagrant-proxyconf` by running the following command (don't run this from the directory containing this project's `Vagrantfile` or it'll fail without performing the plugin downgrade):

```bash
vagrant plugin install --plugin-version 2.0.1 vagrant-proxyconf
```

After the plugin has been updated try re-provisioning the VM (from this project's root directory):

```bash
vagrant destroy && vagrant up
```

**Comment by freemanjp** [comment]
@Igonato this issue is fixed in the [2.13.0](https://github.com/gantsign/development-environment/releases/tag/2.13.0) release of this project.

**Comment by Igonato** [comment]
Nice! I'll try it later on the weekend.

Cool project btw and quick response 🔥 💯 👍
